python/paddle/fluid/tests/unittests/test_reduce_op.py

```diff
@@ -261,6 +266,7 @@ class TestMaxOp_ZeroDim(OpTest):
 
     def setUp(self):
         self.op_type = "reduce_max"
+        self.prim_op_type = "prim"
         self.python_api = paddle.max
         self.inputs = {'X': np.random.random([]).astype("float64")}
         self.attrs = {'dim': []}
```

Comment:
it's better add new op test to check keep_dim = True and FP32

Reply:
> it's better add new op test to check keep_dim = True and FP32

done